.gitignore

```diff
@@ -0,0 +1,17 @@
+*~
+
+*.egg-info
+*.pyc
+*.pyo
+
+.project
+.pydevproject
+/.settings
+/.vscode
+
+/build
+/dist
+/tags
+
+# OSX generated files
+.DS_Store
```

Comment:
Missing line break at the end of file. Editors usually allow you to add end of file line breaks automatically. Maybe Eclipse does it too ?